boards/arm/efm32pg_stk3402a/doc/efm32wg_stk3800.rst

```diff
@@ -0,0 +1,168 @@
+
+Other hardware features are currently not supported by the port.
+
+Connections and IOs
+===================
+
+The EFM32PG SoC has six gpio controllers (PORTA to PORTF), but only four are
+currently enabled (PORTA, PORTB, PORTD and PORTF) for the EFM32PG-STK3402A board.
+
+In the following table, the column Name contains Pin names. For example, PE2
+means Pin number 2 on PORTE, as used in the board's datasheets and manuals.
+
++-------+-------------+-------------------------------------+
+| Name  | Function    | Usage                               |
++=======+=============+=====================================+
+| PF4   | GPIO        | LED0                                |
++-------+-------------+-------------------------------------+
+| PF5   | GPIO        | LED1                                |
++-------+-------------+-------------------------------------+
+| PF6   | GPIO        | Push Button PB0                     |
++-------+-------------+-------------------------------------+
+| PF7   | GPIO        | Push Button PB1                     |
++-------+-------------+-------------------------------------+
+| PA5   | GPIO        | Board Controller Enable             |
+|       |             | EFM_BC_EN                           |
++-------+-------------+-------------------------------------+
+| PD10  | USART0_TX   | UART Console U0_TX #18              |
++-------+-------------+-------------------------------------+
+| PD11  | USART0_RX   | UART Console U0_RX #18              |
++-------+-------------+-------------------------------------+
+
+System Clock
+============
+
+The EFM32PG SoC is configured to use the 40 MHz external oscillator on the
+board.
+
+Serial Port
+===========
+
+The EFM32PG SoC has four USARTs and one Low Energy UART (LEUART).
+
+Programming and Debugging
+*************************
+
+.. note::
+   Before using the kit the first time, you should update the J-Link firmware
+   from `J-Link-Downloads`_
+
+Flashing
+========
+
+The EFM32PG-STK3402A includes an `J-Link`_ serial and debug adaptor built into the
+board. The adaptor provides:
+
+- A USB connection to the host computer, which exposes a Mass Storage and a
+  USB Serial Port.
+- A Serial Flash device, which implements the USB flash disk file storage.
+- A physical UART connection which is relayed over interface USB Serial port.
+
+Flashing an application to EFM32PG-STK3402A
+----------------------------------------
+
+The sample application :ref:`hello_world` is used for this example.
+Build the Zephyr kernel and application:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: efm32pg_stk3402a
+   :goals: build
+
+Connect the EFM32PG-STK3402A to your host computer using the USB port and you
+should see a USB connection which exposes a Mass Storage (STK3402A) and a
+USB Serial Port. Copy the generated zephyr.bin in the STK3402A drive.
+
+Use for example a converter like FT232/CP2102 to connect to UART on the
```

Comment:
Change to:

    Use a USB-to-UART converter such as an FT232/CP2102 to connect to the UART on the
    expansion header.

Reply:
Done